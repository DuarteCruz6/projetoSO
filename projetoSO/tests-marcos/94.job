WRITE [(n4,73)]
DELETE [n4]
WRITE [(j9,54)]
WRITE [(x4,97)(a4,73)(x6,5)(x1,89)]
DELETE [j9]
WAIT 2000
READ [x6]
WRITE [(b8,28)(r3,74)(y5,78)(k9,25)]
WAIT 1000
WRITE [(v2,96)(v4,49)(m3,19)]
BACKUP
WRITE [(q0,33)(n6,90)(d5,94)(t4,73)]
READ [q0]
SHOW
READ [y5]
WRITE [(m2,71)]
SHOW
DELETE [x1,m2,a4,x6,x4]
DELETE [k9,y5]
DELETE [d5,v2,v4,n6]
WAIT 1000
WAIT 2000
DELETE [q0]
WRITE [(c5,67)(j3,83)(s4,99)(y0,82)(x5,23)]
READ [y0]
DELETE [y0,c5,t4,s4]
WRITE [(r0,4)]
READ [r0]
DELETE [r0,x5,j3,m3,r3]
WRITE [(k9,47)(f1,6)(m7,96)]
READ [f1]
READ [f1]
READ [f1]
READ [m7]
SHOW
READ [b8]
WRITE [(g2,66)(y2,9)]
WRITE [(f3,16)(d4,31)(r5,33)(u6,49)]
BACKUP
SHOW
WRITE [(u3,82)(n0,5)(c8,51)(v5,11)(u2,85)]
READ [k9]
BACKUP
DELETE [v5,m7]
WRITE [(m1,48)(h4,32)(j0,100)(z2,65)(k8,45)]
WAIT 3000
WRITE [(n9,15)(r1,42)(k5,66)]
BACKUP
READ [f1]
WRITE [(p6,3)(s3,72)(f9,1)]
WAIT 2000
WRITE [(k6,73)(m9,17)(o7,83)(x0,68)]
WRITE [(i0,17)(f8,18)(j2,12)(z6,52)(a9,50)]
READ [s3]
WRITE [(a6,8)(w0,44)(w0,54)(d6,7)]
WRITE [(e9,83)(o0,89)(i3,75)]
DELETE [n9]
WRITE [(w4,42)(q2,44)(s9,62)(a3,75)(q9,62)]
WRITE [(p3,35)(j5,18)(n2,82)(w7,64)(k3,31)]
WRITE [(j9,0)(o5,41)(s6,39)(v5,89)]
WRITE [(l3,55)(r6,5)]
READ [o0]
WRITE [(h6,28)]
WRITE [(v4,49)]
READ [r1]
WRITE [(n1,6)(g5,28)]
WAIT 3000
WRITE [(t0,9)(a2,40)(d5,66)]
WRITE [(s7,13)(g8,45)(j3,0)(m6,51)]
WRITE [(x1,59)]
WRITE [(h7,99)(i9,45)(v0,30)(p5,69)(l8,55)]
SHOW
READ [f1]
READ [w0]
READ [q9]
DELETE [r1,k8,s9]
DELETE [n1,j5,m9,x1]
READ [w7]
WRITE [(e4,11)]
BACKUP
READ [p3]
WRITE [(y3,39)(r0,82)(o1,4)(t3,79)]
DELETE [f3,h4,o0,f8]
WRITE [(y5,39)]
WRITE [(p9,84)(h5,90)(l4,3)(h0,93)]
