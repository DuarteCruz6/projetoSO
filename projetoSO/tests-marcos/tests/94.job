WRITE [(t0,77)(a2,23)]
WRITE [(g4,19)(z6,30)(n7,83)(p4,96)]
WRITE [(p2,12)]
WRITE [(q6,27)(c4,98)(w5,76)(s6,68)]
READ [n7]
READ [w5]
WRITE [(r7,100)(j4,79)(v9,24)]
WRITE [(e4,75)(h7,89)(e9,19)]
WRITE [(n8,31)(z8,62)(o8,37)(a1,75)]
DELETE [a1]
WRITE [(d7,21)(d5,77)]
WRITE [(q9,88)(b6,66)(a4,73)(r1,75)(z9,50)]
BACKUP
WRITE [(b5,38)]
WRITE [(x4,26)(p5,78)(n4,12)]
WRITE [(i6,60)(v4,3)(j2,100)(g5,77)(j9,47)]
DELETE [s6,t0,n4,q9,h7]
WRITE [(i0,38)(p7,10)(a8,54)]
WRITE [(w6,36)(d8,71)(n3,38)]
READ [b6]
SHOW
READ [p2]
WRITE [(f3,45)(a3,96)(w4,92)]
BACKUP
WRITE [(v1,100)(o3,71)(u4,42)]
READ [w5]
READ [a4]
WRITE [(v2,8)(f6,5)(y8,87)]
WRITE [(r3,33)]
WRITE [(v6,38)(f8,80)(y7,32)(b4,11)]
WRITE [(h9,37)(h7,25)]
WAIT 1000
WAIT 1000
READ [d5]
WAIT 2000
DELETE [d8,v6,i6,n8]
DELETE [h7,a4]
WRITE [(y3,96)(d8,84)]
READ [y7]
WRITE [(g2,92)(r6,51)(m6,22)]
READ [n7]
WRITE [(f1,3)(e3,42)]
DELETE [w6,d8]
WRITE [(y4,50)]
SHOW
WRITE [(s6,74)(k3,68)]
READ [z6]
WRITE [(l1,57)]
WRITE [(p0,69)]
WRITE [(f4,4)(e1,88)(k5,59)(s2,35)]
BACKUP
WRITE [(t2,78)(l6,64)(a6,64)(f7,19)]
READ [t2]
READ [v9]
WRITE [(j0,77)(u3,56)(y2,82)(l9,87)]
READ [a8]
DELETE [g5,v2,a8,e4]
READ [i0]
WRITE [(x6,91)(o7,27)]
SHOW
WRITE [(w8,26)(z4,42)(y0,15)(r2,83)(n5,7)]
READ [y8]
WRITE [(q8,85)(u9,87)(k0,3)]
WAIT 1000
BACKUP
WRITE [(x8,86)(i9,7)]
WRITE [(m4,17)(t7,89)(m3,75)(s8,27)(q3,67)]
WRITE [(c7,34)(k2,44)]
WRITE [(y6,97)(h2,80)(o9,30)(o4,67)]
WRITE [(t0,96)(v2,50)(z7,88)(m0,3)]
DELETE [f8,v9]
SHOW
WAIT 2000
DELETE [h9]
READ [z6]
DELETE [d7,q8,n3]
DELETE [j4,y4,i9]
DELETE [o4]
WRITE [(h3,93)(d9,21)(e2,78)]
WRITE [(h6,61)(g1,31)]
WAIT 3000
DELETE [z8,t2,h3]
BACKUP
WRITE [(w6,29)(d3,55)]
WRITE [(x3,42)(w7,94)]
READ [y7]
WRITE [(d7,12)]
WRITE [(c1,89)(r5,15)(d4,93)(a4,79)(e7,63)]
WRITE [(f8,37)]
