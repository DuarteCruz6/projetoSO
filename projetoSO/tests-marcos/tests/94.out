[(n7,83)]
[(w5,76)]
[(b6,66)]
(a8, 54)
(a4, 73)
(a2, 23)
(b5, 38)
(b6, 66)
(c4, 98)
(d8, 71)
(d5, 77)
(d7, 21)
(e9, 19)
(e4, 75)
(g5, 77)
(g4, 19)
(i0, 38)
(i6, 60)
(j9, 47)
(j2, 100)
(j4, 79)
(n3, 38)
(n8, 31)
(n7, 83)
(o8, 37)
(p7, 10)
(p5, 78)
(p2, 12)
(p4, 96)
(q6, 27)
(r1, 75)
(r7, 100)
(v4, 3)
(v9, 24)
(w6, 36)
(w5, 76)
(x4, 26)
(z9, 50)
(z8, 62)
(z6, 30)
[(p2,12)]
[(w5,76)]
[(a4,73)]
[(d5,77)]
[(y7,32)]
[(n7,83)]
(a3, 96)
(a8, 54)
(a2, 23)
(b4, 11)
(b5, 38)
(b6, 66)
(c4, 98)
(d5, 77)
(d7, 21)
(e3, 42)
(e9, 19)
(e4, 75)
(f1, 3)
(f8, 80)
(f6, 5)
(f3, 45)
(g2, 92)
(g5, 77)
(g4, 19)
(h9, 37)
(i0, 38)
(j9, 47)
(j2, 100)
(j4, 79)
(m6, 22)
(n3, 38)
(n7, 83)
(o3, 71)
(o8, 37)
(p7, 10)
(p5, 78)
(p2, 12)
(p4, 96)
(q6, 27)
(r6, 51)
(r3, 33)
(r1, 75)
(r7, 100)
(u4, 42)
(v2, 8)
(v1, 100)
(v4, 3)
(v9, 24)
(w4, 92)
(w5, 76)
(x4, 26)
(y4, 50)
(y3, 96)
(y7, 32)
(y8, 87)
(z9, 50)
(z8, 62)
(z6, 30)
[(z6,30)]
[(t2,78)]
[(v9,24)]
[(a8,54)]
[(i0,38)]
(a6, 64)
(a3, 96)
(a2, 23)
(b4, 11)
(b5, 38)
(b6, 66)
(c4, 98)
(d5, 77)
(d7, 21)
(e1, 88)
(e3, 42)
(e9, 19)
(f7, 19)
(f4, 4)
(f1, 3)
(f8, 80)
(f6, 5)
(f3, 45)
(g2, 92)
(g4, 19)
(h9, 37)
(i0, 38)
(j0, 77)
(j9, 47)
(j2, 100)
(j4, 79)
(k5, 59)
(k3, 68)
(l9, 87)
(l6, 64)
(l1, 57)
(m6, 22)
(n3, 38)
(n7, 83)
(o7, 27)
(o3, 71)
(o8, 37)
(p0, 69)
(p7, 10)
(p5, 78)
(p2, 12)
(p4, 96)
(q6, 27)
(r6, 51)
(r3, 33)
(r1, 75)
(r7, 100)
(s2, 35)
(s6, 74)
(t2, 78)
(u3, 56)
(u4, 42)
(v1, 100)
(v4, 3)
(v9, 24)
(w4, 92)
(w5, 76)
(x6, 91)
(x4, 26)
(y2, 82)
(y4, 50)
(y3, 96)
(y7, 32)
(y8, 87)
(z9, 50)
(z8, 62)
(z6, 30)
[(y8,87)]
(a6, 64)
(a3, 96)
(a2, 23)
(b4, 11)
(b5, 38)
(b6, 66)
(c7, 34)
(c4, 98)
(d5, 77)
(d7, 21)
(e1, 88)
(e3, 42)
(e9, 19)
(f7, 19)
(f4, 4)
(f1, 3)
(f6, 5)
(f3, 45)
(g2, 92)
(g4, 19)
(h2, 80)
(h9, 37)
(i9, 7)
(i0, 38)
(j0, 77)
(j9, 47)
(j2, 100)
(j4, 79)
(k2, 44)
(k0, 3)
(k5, 59)
(k3, 68)
(l9, 87)
(l6, 64)
(l1, 57)
(m0, 3)
(m3, 75)
(m4, 17)
(m6, 22)
(n5, 7)
(n3, 38)
(n7, 83)
(o4, 67)
(o9, 30)
(o7, 27)
(o3, 71)
(o8, 37)
(p0, 69)
(p7, 10)
(p5, 78)
(p2, 12)
(p4, 96)
(q3, 67)
(q8, 85)
(q6, 27)
(r2, 83)
(r6, 51)
(r3, 33)
(r1, 75)
(r7, 100)
(s8, 27)
(s2, 35)
(s6, 74)
(t0, 96)
(t7, 89)
(t2, 78)
(u9, 87)
(u3, 56)
(u4, 42)
(v2, 50)
(v1, 100)
(v4, 3)
(w8, 26)
(w4, 92)
(w5, 76)
(x8, 86)
(x6, 91)
(x4, 26)
(y6, 97)
(y0, 15)
(y2, 82)
(y4, 50)
(y3, 96)
(y7, 32)
(y8, 87)
(z7, 88)
(z4, 42)
(z9, 50)
(z8, 62)
(z6, 30)
[(z6,30)]
[(y7,32)]
